test/integration-test-common.sh

```diff
@@ -15,8 +15,8 @@ fi
 chmod 600 "$S3FS_CREDENTIALS_FILE"
 
 S3PROXY_VERSION="1.4.0"
-S3PROXY_BINARY="s3proxy-${S3PROXY_VERSION}"
-if [ ! -e "${S3PROXY_BINARY}" ]; then
+S3PROXY_BINARY=${S3PROXY_BINARY-"s3proxy-${S3PROXY_VERSION}"}
+if [ -n "${S3PROXY_BINARY}" ] && [ ! -e "${S3PROXY_BINARY}" ]; then
     wget "https://github.com/andrewgaul/s3proxy/releases/download/s3proxy-${S3PROXY_VERSION}/s3proxy" \
```

Comment:
@andrewgaul I just pushed an amended commit that fixed the new logic here.  Sorry about that.  It works now if s3proxy isn't already downloaded.